swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        self.id = id if id is not None else uuid.uuid4().int
+        self.condition = condition
+
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+
+    
+    def get_category(self):
+        return "Item"
+        # class_name = __class__.__name__
```

Comment:
Instead of hardcoding the string "Item" we should be returning the class name like you have on line 14. From the instance of `self` we can get that instance's class name:

```python
def get_category(self):
        return self.__class__.__name__
```

What if we decided to refactor the Item class and renamed it SwapItem? Your get_category would not return the correct value since we are always returning "Item" instead of dynamically grabbing the value from the instance.